### PR TITLE
Create make 

### PR DESCRIPTION
**Pull request by mariusjagminas**
mod/make-file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+build.zip:
```
Comment (kowalski) [inline Makefile:1]: You could add dependencies here analogically to how we set them in Makefile in infrastrucutre repo

**Review by kowalski — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (kowalski) [reply Makefile:1]: hold on, this target should be called `users-service.zip`, because the file produced by the command below is `users-service.zip`.

Comment (mariusjagminas) [reply Makefile:1]: or it could be build-user-service.zip ?

Comment (kowalski) [reply Makefile:1]: no

Comment (kowalski) [reply Makefile:1]: the name of the target must correspond to the name of the file the target produces
that's the only way that makes dependency system work

Comment (mariusjagminas) [reply Makefile:1]: ok

**Review by kowalski — APPROVED** [review]

**Pull request merged**